apps/src/codebridge/Console/ControlButtons.tsx

```diff
@@ -54,15 +57,14 @@ const ControlButtons: React.FunctionComponent = () => {
 
   useLifecycleNotifier(LifecycleEvent.LevelLoadCompleted, resetStatus);
 
-  const handleRun = (runTests: boolean) => {
```

Comment:
we weren't ever calling this with `runTests` set to true anymore, so I removed the boolean, since we are now logging this as a run click specifically.